src/aprl/envs/multi_agent.py

```diff
@@ -362,21 +362,20 @@ def step_async(self, actions):
 
     def step_wait(self):
         observations, rewards, self._dones, infos = self.venv.step_wait()
-        observations = self._get_augmented_obs(observations)
+        observations = self._get_updated_obs(observations)
         return observations, rewards, self._dones, infos
 
     def reset(self):
         observations = self.venv.reset()
-        observations = self._get_augmented_obs(observations)
+        observations = self._get_updated_obs(observations)
         return observations
 
-    def _get_augmented_obs(self, observations):
+    def _get_updated_obs(self, observations):
```

Comment:
Yeah it was because at one point I wanted to have a convention for all VecEnvWrappers where they would have a  `_get_updated_obs` method but I decided not to. Forgot to revert this back, but I'm going to, since `_get_augmented_obs` is more descriptive.